tests/unit/broccoli/ember-app/import-test.js

```diff
@@ -0,0 +1,221 @@
+'use strict';
+
+const co = require('co');
+const fs = require('fs-extra');
+const broccoliTestHelper = require('broccoli-test-helper');
+const expect = require('chai').expect;
+
+const EmberApp = require('../../../../lib/broccoli/ember-app');
+const MockCLI = require('../../../helpers/mock-cli');
+const Project = require('../../../../lib/models/project');
+
+const buildOutput = broccoliTestHelper.buildOutput;
+const createTempDir = broccoliTestHelper.createTempDir;
+
+describe('EmberApp.import()', function() {
+  let input, cwd;
+
+  beforeEach(function() {
+    cwd = process.cwd();
+    return createTempDir().then(tempDir => (input = tempDir));
+  });
+
+  afterEach(function() {
+    process.chdir(cwd);
+    delete process.env.EMBER_ENV;
+    return input.dispose();
+  });
+
+  function createApp(options) {
+    options = options || {};
+
+    let path = input.path();
+    process.chdir(path);
+
+    let pkg = fs.readJsonSync(`${path}/package.json`);
+
+    let cli = new MockCLI();
+    let project = new Project(path, pkg, cli.ui, cli);
+
+    let app = new EmberApp({
+      project,
+      _ignoreMissingLoader: true,
+      sourcemaps: { enabled: false },
+    }, options);
+
+    app._processedTemplatesTree = () => '';
+
+    return app;
+  }
+
+  it('does not import bower dependencies if they are not explicitly imported', co.wrap(function *() {
+    input.copy(`${__dirname}/../../../fixtures/new-app`);
```

Comment:
We shouldn't embed a full `ember new foo --skip-npm --skip-bower` into `tests/fixtures/`. If we need that (and it seems fine to generally have it), then we should generate it (the generator doesn't take long to run) or clone it from `ember-new-output`.